frappe/website/doctype/top_bar_item/top_bar_item.json

```diff
@@ -63,12 +55,20 @@
    "fieldname": "icon",
    "fieldtype": "Attach Image",
    "label": "Icon"
+  },
+  {
+   "default": "0",
+   "depends_on": "eval:doc.url",
+   "fieldname": "in_new_tab",
```

Comment:
Why not go all the way and call it `open_in_new_tab`

Reply:
Okay, will do.

Reply:
**Update:** done!